snapshots/irena/2023-12-12/renewable_electricity_capacity_and_generation.xlsm.dvc

```diff
@@ -1,48 +1,55 @@
-# TODO: This file is a duplicate of the previous step. It is not yet used in the dag and should be updated soon.
-
 meta:
-  name: Renewable Electricity Capacity and Generation Statistics
-  publication_year: 2023
+  origin:
+    # Data product / Snapshot
+    title: Renewable Electricity Capacity and Generation Statistics
+    date_published: 2023-03-01
+    description: |-
+      IRENA publishes detailed statistics on renewable energy capacity, power generation and renewable energy balances. This data is collected directly from members using the IRENA Renewable Energy Statistics questionnaire and is also supplemented by desk research where official statistics are not available.
+
+      Some technologies include others, following this schema:
+
+      Total renewable energy
+        1. Hydropower
+            1.1 Renewable hydropower
+            1.2 Pumped storage *
+        2. Marine
+        3. Wind
+            3.1 Onshore wind energy
+            3.2 Offshore wind energy
+        4. Solar
+            4.1 Solar photovoltaic
+            4.2 Concentrated solar power
+        5. Bioenergy
+            5.1 Solid biofuels
+                5.1.1 Bagasse
+                5.1.2 Renewable municipal waste
+                5.1.3 Other solid biofuels
+            5.2 Liquid biofuels
+            5.3 Biogas
+        6. Geothermal
 
-  publication_date: '2023-03-01'
-  source_name: International Renewable Energy Agency (IRENA)
-  source_published_by: International Renewable Energy Agency © IRENA 2022
-  url: https://www.irena.org/Statistics/Download-query-tools
-  source_data_url:
-    https://www.irena.org/IRENADocuments/IRENA_RE_electricity_statistics_-_Query_tool.xlsm
-  license_url:
-    https://www.irena.org/-/media/Files/IRENA/Agency/Publication/2023/Mar/IRENA_RE_Capacity_Statistics_2023.pdf
-  license_name: Copyright © IRENA 2023
-  date_accessed: 2023-06-28
-  is_public: true
-  description: |
-    IRENA publishes detailed statistics on renewable energy capacity, power generation and renewable energy balances. This data is collected directly from members using the IRENA Renewable Energy Statistics questionnaire and is also supplemented by desk research where official statistics are not available.
+      * Pumped storage is included under the "Hydropower" category but not in the "Total renewable energy". Generation from mixed plants is split between hydropower and pumped storage as appropriate.
 
-    Some technologies include others, following this schema:
+    # Citation
+    producer: International Renewable Energy Agency
+    citation_full: |-
```

Comment:
is there any reason to include all of this information in the citation full? the only recommended citation I can see is - `IRENA (2023), Renewable capacity statistics 2023, International Renewable Energy Agency, Abu Dhabi.`